### ISSUE TITLE
Mapping error for sfRenderTexture_createWithSettings (CSFML 2.5.0)

### ISSUE THREAD

**Issue by olivier-spinelli** [SFML/SFML.Net#162]
The RenderTexture should use the new ctor with the ContextSettings:
 
```
[DllImport( CSFML.Graphics, CallingConvention = CallingConvention.Cdecl ), SuppressUnmanagedCodeSecurity]
static extern IntPtr sfRenderTexture_createWithSettings( uint Width, uint Height, ContextSettings Settings );
```
Unfortunately, sfRenderTexture_createWithSettings is not exposed in CSFML 2.5.0 native.
To easily exhibit the error, redefine the default ctor implentation of RenderTexture like it SHOULD be (since the one with bool depthBuffer is deprecated):
```
        public RenderTexture(uint width, uint height) :
            this(width, height, default(ContextSettings))
        {
        }
```
And runs the shader.csproj sample. Kaboom!!

And I don't understand why since it is actually part of the C code base:
https://github.com/SFML/CSFML/blob/282621e4995b5a22c4c5981e8f2cad98807d3554/include/SFML/Graphics/RenderTexture.h#L68

**Comment by DemoXinMC** [comment]
RenderTexture(uint, uint) is corrected in #167 to use a default constructed ContextSettings and the signature for sfRenderTexture_createFromSettings has had its signature updated to match CSFML

**Comment by LaurentGomila** [comment]
Fixed by #168
